castle/cms/behaviors/configure.zcml

```diff
@@ -68,5 +68,25 @@
     provides=".adjustablefont.IAdjustableFontSizeQueryListing"
     for="plone.dexterity.interfaces.IDexterityContent"
     />
+  <plone:behavior
+    name="castle.versioning"
+    title="Versioning"
+    description="Versioning support with CMFEditions"
+    provides=".versionable.IVersionable"
+    factory=".versionable.Versionable"
+    marker=".versionable.IVersioningSupport"
+    for="plone.dexterity.interfaces.IDexterityContent"
+    />
+  <subscriber
```

Comment:
can these two <subscriber> entries be placed in castle/cms/subscribers.zcml? Also, since it looks like they're exact copies of the methods in plone.app.versioningbehavior, and I don't see anywhere where you've removed the import of plone.app.versioningbehavior, couldn't you just either a) have the handler point to the method from plone.app.versioningbehavior (if that works that way in the zcml, or if not b)in our subscribers.py, just import the two methods from plone.app.versioningbehavior, and have the zcml point to the imported methods in subscribers? It seems like it'd save a bunch of castle code if that could be done. Thoughs?